packages/commercetools/theme/middleware/checkout.js

```diff
@@ -0,0 +1,37 @@
+import { getMe } from '@vue-storefront/commercetools-api';
+
+const canEnterPersonalDetails = cart => !cart.customerId;
+
+const canEnterShipping = cart => cart.customerEmail || cart.customerId;
+
+const canEnterPayment = cart => cart.shippingInfo && cart.shippingAddress;
+
+const canEnterReview = cart => cart.billingAddress;
```

Comment:
name of the function suggests it will return a boolean type. In this case, an object is returned